changes/6836.fixed

```diff
@@ -0,0 +1 @@
+Fixed Component Template models incorrectly assume `Notes` support in the API.
```

Comment:
Looks like there's a number of other APIs that were also fixed (extras, ipam, wireless through tables), would be good to document that as well.